app/Http/Controllers/ApiPhotosController.php

```diff
@@ -289,7 +289,11 @@ public function storeWithTags (Request $request) :array
 
         $userId = Auth::guard('api')->user()->id;
 
-        dispatch (new AddTags($userId, $photo->id, $request->tags));
+        dispatch (new AddTags(
+            $userId,
+            $photo->id,
+            json_decode($request->tags, true)
```

Comment:
What if people are using older version of app and do not update it? This will fail?

Reply:
This is the new route `storeWithTags`, which older app versions don't even know exists, this will be added in the latest versions.

Reply:
I see now, thanks. Comments could be updated when possible